docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,381 @@
+
+#### Import an existing project
+
+To import existing Dagster code, you can use Dagster's GitHub app or the dagster-cloud CLI.
+
+<TabGroup>
+<TabItem name="GitHub integration">
+
+**Using the GitHub integration**
+
+Using the GitHub integration to import an existing GitHub repository also sets up CI/CD for you.
+
+Before you get started, note that the repository must have a `dagster_cloud.yaml` file in order for Dagster Cloud to deploy it. This file defines the [code locations](/concepts/code-locations) in your Dagster code.
+
+If this file doesn't currently exist, create it in the root of your repository with the following code:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: my_location_name # the name of the code location
+    code_source:
+      package_name: hackernews # the name of the python package associated with the code location
+```
+
+After you've committed the file to the repository, come back to Dagster Cloud to complete the import process:
+
+1. Click the **Import an existing project** tab.
+2. Sign in to your GitHub account, if prompted.
+3. In Dagster Cloud, define the following:
+   - **Git scope** - Select the organization or account that the repository is in.
+   - **Repository** - Select the repository.
+4. Click **Deploy**.
+
+When finished, [continue to the next step](#step-4-set-up-environment-variables-and-secrets).
+
+</TabItem>
+<TabItem name="dagster-cloud CLI">
+
+**Using the dagster-cloud CLI**
+
+<Note>
+  Using the dagster-cloud CLI requires a recent version of Python 3 and Docker.
+</Note>
+
+To complete this step using the CLI, you can use your own Dagster code or the [Dagster starter kit](https://github.com/dagster-io/quickstart-etl). The starter kit is a template with everything you need to get started using Serverless in Dagster Cloud, including CI/CD configuration and the required `dagster_cloud.yaml` file.
+
+- **If using the template**, [clone the repository](https://github.com/dagster-io/quickstart-etl) to your local environment.
+
+- **If not using the template**, add `dagster-cloud` as a dependency in `setup.py`. \[Click here for an example]\(<https://github.com/dagster-io/quickstart-etl/blob/main/setup.py>. This is already done for you if using the starter kit.
+
+After you've finished setting up your local project, move on to deploying using the CLI:
+
+1. To install the dagster-cloud CLI, run:
+
+   ```shell
+   pip install dagster-cloud
+   ```
+
+2. Next, you'll need to authenticate to Dagster Cloud. Run the following command and follow the prompts to log in:
+
+   ```shell
+   dagster-cloud config setup
+   ```
+
+3. After you've successfully authenticated, run the following to deploy your code to Dagster Cloud:
+
+   ```shell
+   dagster-cloud serverless deploy \
+         --location-name "<LOCATION_NAME>" \       # name of the code loation
+         --package-name <PACKAGE_NAME> \           # name of the Python package associated with the code location
```

Comment:
nit but one of these has quotes and the other does not - they're both strings so can be consistent i think